drivers/scale_host.go

```diff
@@ -44,8 +44,21 @@ func (s *ScaleHostDriver) ValidatePayload(conf interface{}, apiClient *client.Ra
 		return http.StatusBadRequest, fmt.Errorf("Invalid amount: %v", config.Amount)
 	}
 
-	if config.HostSelector == nil {
-		return http.StatusBadRequest, fmt.Errorf("HostSelector not provided")
+	if config.HostSelector == nil && config.HostTemplateID == "" {
+		return http.StatusBadRequest, fmt.Errorf("HostSelector or HostTemplateID is not provided")
```

Comment:
Still keep the hostTempllateID == "" check, just don't mention it in th error message